Automattic/wp-calypso
# Marketplace: Add category to Plugin page title

Fixes https://github.com/Automattic/wp-calypso/issues/76010

## Proposed Changes

* Adds the category to the `document.title` in Plugins screen when there is a category selected

|Before | After|
|-------|------|
|![CleanShot 2023-04-28 at 13 16 42@2x](https://user-images.githubusercontent.com/3519124/235134303-8c5eadfc-7adf-4e28-9608-606479f936c9.png) | ![CleanShot 2023-04-28 at 12 33 30@2x](https://user-images.githubusercontent.com/3519124/235132988-3386fb9f-6f6f-42ea-b9f7-be0e896516da.png) |


## Testing Instructions



* Apply this patch
* Navigate to `/plugins` page and select different categories
* Hover over the Tab to see the title of the page
* Confirm it contains the selected category
* Confirm that this works for both logged-in pages and logged-out pages. To test logged-out pages, the browser can be used in Incognito mode.

## Pre-merge Checklist
